### ISSUE TITLE
[Bug]: expo doctor reports @expo/config-plugins incompatibility with Expo 47

### ISSUE THREAD

**Issue by pwfcurry** [braze-inc/braze-expo-plugin#8]
### Which Platforms?

Both

### Which React Native Version?

0.70.5

### Which @braze/expo-plugin version?

0.5.0

### Which react-native-appboy-sdk SDK version?

1.41.0

### Repro Rate

100% of the time

### Steps To Reproduce

When using Expo 47, run `expo doctor`.

```
> expo doctor
Expected package @expo/config-plugins@^5.0.2
Found invalid:
  @expo/config-plugins@4.1.5
  (for more info, run: npm why @expo/config-plugins)
```

### Expected Behavior

`expo doctor` passes

### Actual Incorrect Behavior

`expo doctor` fails

### Verbose Logs

```shell
> yarn why @expo/config-plugins
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "@expo/config-plugins"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@expo/config-plugins@5.0.4"
info Has been hoisted to "@expo/config-plugins"
info Reasons this module exists
   - Hoisted from "expo#@expo#config-plugins"
   - Hoisted from "expo-updates#@expo#config-plugins"
   - Hoisted from "expo#@expo#cli#@expo#config-plugins"
   - Hoisted from "expo#@expo#config#@expo#config-plugins"
   - Hoisted from "expo-splash-screen#@expo#prebuild-config#@expo#config-plugins"
info Disk size without dependencies: "1.98MB"
info Disk size with unique dependencies: "3.63MB"
info Disk size with transitive dependencies: "7.39MB"
info Number of shared dependencies: 46
=> Found "@braze/expo-plugin#@expo/config-plugins@4.1.5"
info This module exists because "@braze#expo-plugin" depends on it.
info Disk size without dependencies: "1.6MB"
info Disk size with unique dependencies: "3.25MB"
info Disk size with transitive dependencies: "7.02MB"
info Number of shared dependencies: 46
✨  Done in 1.06s.
```

### Additional Information

This doesn't appear to be causing any issues, other than `expo doctor` failing, but we've yet to use the braze sdk in anger.

One option is to pin the dependency via a resolution, but we'd rather avoid that.

**Comment by wesleyorbin** [comment]
Hi @pwfcurry. Thanks for bringing this to our attention. It will be addressed in the next release.

**Comment by pwfcurry** [comment]
Looks like this is fixed as of 0.6.0 🎉
